mono/metadata/profiler.c

```diff
@@ -203,6 +203,9 @@ mono_profiler_get_coverage_data (MonoProfilerHandle handle, MonoMethod *method,
 	if (!mono_profiler_state.code_coverage)
 		return FALSE;
 
+	if ((method->flags & METHOD_ATTRIBUTE_ABSTRACT) || (method->iflags & METHOD_IMPL_ATTRIBUTE_RUNTIME) || (method->iflags & METHOD_IMPL_ATTRIBUTE_INTERNAL_CALL) || (method->flags & METHOD_ATTRIBUTE_PINVOKE_IMPL))
```

Comment:
I'd prefer to write it like this:

```c
	if ((method->flags & (METHOD_ATTRIBUTE_ABSTRACT | METHOD_ATTRIBUTE_PINVOKE_IMPL)) ||
	    (method->iflags & (METHOD_IMPL_ATTRIBUTE_RUNTIME | METHOD_IMPL_ATTRIBUTE_INTERNAL_CALL)))
		return FALSE;
```

Just to avoid super long lines.